README.md

```diff
@@ -839,67 +839,73 @@ percona-scheduler-admin usage info
-  --remove-all-servers               When used with --update-cluster, this will remove all
-                                     servers belonging to the current cluster before
-                                     updating the list.
-  --server=<IPADDRESS>:<PORT>        Specifies the IP address and port for a single server. This can
-                                     be used with --syncusers or --sync-multi-cluster-users
-                                     to sync a single non-cluster server node.
-  --add-query-rule                   Create query rules for synced mysql user. This is applicable only
-                                     for singlewrite mode and works only with --syncusers
-                                     and --sync-multi-cluster-users options.
-  --force                            This option will skip existing configuration checks in mysql_servers,
-                                     mysql_users and mysql_galera_hostgroups tables. This option will
-                                     work with '--enable' and '--update-cluster'.
-                                     This will also cause certain checks to issue warnings instead
-                                     of an error.
+  --debug                            Enables additional debug logging.
+  --disable, -d                      Remove any Percona XtraDB Cluster configurations from ProxySQL
   --disable-updates                  Disable admin updates for ProxySQL cluster for the
                                      current operation. The default is to not change the
                                      admin variable settings.  If this option is specifed,
                                      these options will be set to false.
                                      (default: updates are not disabled)
-  --use-stdin-for-credentials        If set, then the MySQL client will use stdin to send credentials
-                                     to the client (instead of process substition).
-                                     (default: process subsitution is used)
-  --trace                            Enables shell-level tracing for this shell script
-  --debug                            Enables additional debug logging.
-  --help                             Displays this help text.
-
-These options are the possible operations for percona-scheduler-admin.
-One of the options below must be provided.
-  --adduser                          Adds the Percona XtraDB Cluster application user to the ProxySQL database
-  --disable, -d                      Remove any Percona XtraDB Cluster configurations from ProxySQL
   --enable, -e                       Auto-configure Percona XtraDB Cluster nodes into ProxySQL
+  --help                             Displays this help text.
+  --is-enabled                       Checks if the current configuration is enabled in ProxySQL.
+  --server=<IPADDRESS>:<PORT>        Specifies the IP address and port for a single server. This can
```

Comment:
This option can be
       used with ``--syncusers`` or ``sync-multi-cluster-users`` to sync a
       single non-cluster server node but does not require these options.

Reply:
Thanks for pointing this out. This is actually a dependent option (needs --syncusers or --sync-multi-cluster-users) and cannot be used independently.

Changed to.

```
   --server=<IPADDRESS>:<PORT>        This option can be used with --syncusers or 
     ┆   ┆   ┆   ┆   ┆   ┆   ┆   ┆   ┆--sync-multi-cluster-users to sync a single non-cluster server
                                      node.
```